frontend/common/src/fakeData/fakeExperiences.ts

```diff
@@ -58,6 +58,11 @@ const sampleSkill2: Skill = {
   name: theSkillString2,
   experienceSkillRecord: theExperienceSkillRecord,
 };
+function formattedDeate(date: string) {
```

Comment:
Just going to be a bit picky, I assume you meant _formattedDate_

Reply:
you are right!